src/pet.js

```diff
@@ -0,0 +1,87 @@
+// INITIAL PROPERTY VALUES
+const initialAge = 0;
+const initialHunger = 0;
+const initialFitness = 10;
+// DEATH THRESHOLDS
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0;
+const MAXIMUM_HUNGER = 10;
+const MINIMUM_HUNGER = 0; 
+const MAXIMUM_AGE = 30;
+
+function Pet(name) {
+    this.name = name;
+    this.age = initialAge;
+    this.hunger = initialHunger;
+    this.fitness = initialFitness;
+    this.children = [];
+};
+
+Pet.prototype = {
+    // added a getter to the prototype to dynamically track the state of the pet
+    get isAlive() {
+      return (
+          this.age < MAXIMUM_AGE 
+          && 
+          this.hunger < MAXIMUM_HUNGER 
+          && 
+          this.fitness > MINIMUM_FITNESS);git
+    }
```

Comment:
Could fix the formatting a bit here. Conventionally its one statement per line, with the operator (&&) on the end of the line: 
```suggestion
    get isAlive() {
        return (
            this.age < MAXIMUM_AGE && 
            this.hunger < MAXIMUM_HUNGER && 
            this.fitness > MINIMUM_FITNESS
        )
    }
```
This varies from company to company and is not a hard and fast rule, but it is pretty common